### ISSUE TITLE
Bunyan 0.22.3 crashed: TypeError: Object.keys called on non-object

### ISSUE THREAD

**Issue by jtblin** [trentm/node-bunyan#198]
Hi

As indicated in bunyan cli output, sending details. It happens using [express-bunyan-logger](https://www.npmjs.org/package/express-bunyan-logger).

```
* * *
* platform: darwin
* node version: v0.10.32
* bunyan version: 0.22.3
* argv: ["node","/usr/local/bin/bunyan"]
* log line: "{\"name\":\"express\",\"hostname\":\"jtblin\",\"pid\":77594,\"req_id\":\"2471f5f4-448f-41ed-ab4a-2ded1ad78ecf\",\"level\":30,\"remote-address\":\"127.0.0.1\",\"ip\":\"127.0.0.1\",\"method\":\"GET\",\"url\":\"/\",\"referer\":\"-\",\"user-agent\":{\"family\":\"Chrome\",\"major\":\"39\",\"minor\":\"0\",\"patch\":\"2171\",\"device\":{\"family\":\"Other\",\"major\":\"0\",\"minor\":\"0\",\"patch\":\"0\"},\"os\":{\"family\":\"Mac OS X\",\"major\":\"10\",\"minor\":\"10\",\"patch\":\"1\"}},\"short-body\":\"undefined\",\"http-version\":\"1.1\",\"response-time\":1101,\"status-code\":304,\"req-headers\":{\"host\":\"localhost:8080\",\"connection\":\"keep-alive\",\"cache-control\":\"max-age=0\",\"authorization\":\"Basic\",\"accept\":\"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8\",\"user-agent\":\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.71 Safari/537.36\",\"accept-encoding\":\"gzip, deflate, sdch\",\"accept-language\":\"en-US,en;q=0.8,fr;q=0.6\",\"cookie\":\"micros.token=RHVmNkxpaHYyQVJYOTlKZlZ1UzJjWDJzcFBiS2dTQXM=; _ga=GA1.1.1776647255.1414892961; cookieLequipebis=1; bm_monthly_unique=true; wlsuid=413107142337619523; bm_last_load_status=BLOCKING; _foglio_p={\\\"pmd\\\":{\\\"fver\\\":\\\"20140625-1\\\",\\\"lp_ref\\\":\\\"\\\",\\\"lp_path\\\":\\\"/\\\",\\\"lp_reftype\\\":\\\"direct\\\",\\\"lp_type\\\":\\\"homepage\\\",\\\"u_id\\\":\\\"32e1a301-7564-45dc-9934-d725057081be\\\",\\\"s_id\\\":\\\"c4b9152c-016d-42a5-9ad4-b1c46269e641\\\",\\\"ims\\\":\\\"0\\\"},\\\"pt\\\":{\\\"s_st\\\":1417413115188,\\\"lv\\\":1417413338583},\\\"pm\\\":{\\\"sc_lts\\\":[21,12,8],\\\"sc_sum\\\":42102,\\\"sc_pv\\\":3,\\\"sc_se\\\":0,\\\"sc_apd\\\":0.5,\\\"sc_wapd\\\":0.5,\\\"sc_mean\\\":14034,\\\"sc_med\\\":12000,\\\"pp_nm_loa_e\\\":8257,\\\"sp_mean\\\":16923,\\\"sp_med\\\":17000},\\\"ups\\\":{},\\\"upm\\\":{}}\",\"if-none-match\":\"W/\\\"c-1c291ca3\\\"\"},\"res-headers\":{\"x-powered-by\":\"Express\",\"etag\":\"W/\\\"c-1c291ca3\\\"\"},\"req\":{\"method\":\"GET\",\"url\":\"/\",\"headers\":\"[Circular]\",\"remoteAddress\":\"127.0.0.1\",\"remotePort\":59830},\"res\":{\"statusCode\":304,\"header\":\"HTTP/1.1 304 Not Modified\\r\\nX-Powered-By: Express\\r\\nETag: W/\\\"c-1c291ca3\\\"\\r\\nDate: Thu, 04 Dec 2014 11:47:43 GMT\\r\\nConnection: keep-alive\\r\\n\\r\\n\"},\"incoming\":\"<--\",\"msg\":\"127.0.0.1 <-- GET / HTTP/1.1 304 - - Chrome 39.0 Mac OS X 10.10.1 1101 ms\",\"time\":\"2014-12-04T11:47:43.064Z\",\"v\":0}"
* stack:
*     TypeError: Object.keys called on non-object
*         at Function.keys (native)
*         at emitRecord (/usr/local/lib/node_modules/bunyan/bin/bunyan:761:35)
*         at handleLogLine (/usr/local/lib/node_modules/bunyan/bin/bunyan:650:16)
*         at Socket.<anonymous> (/usr/local/lib/node_modules/bunyan/bin/bunyan:1009:13)
*         at Socket.emit (events.js:95:17)
*         at Socket.<anonymous> (_stream_readable.js:764:14)
*         at Socket.emit (events.js:92:17)
*         at emitReadable_ (_stream_readable.js:426:10)
*         at emitReadable (_stream_readable.js:422:5)
*         at readableAddChunk (_stream_readable.js:165:9)
* * *
```

Thanks for the great lib btw.

**Comment by Raffiki** [comment]
can it be the referenced commit is not yet published onto npm (version 1.2.3 that is)?

I'm doing a git clone for now.

**Comment by trentm** [comment]
@Raffiki This fix is a in bunyan version published to npm.
